pkg/proxy/ipvs/proxier.go

```diff
@@ -451,7 +451,7 @@ func (handle *LinuxKernelHandler) GetModules() ([]string, error) {
 	builtinModsFilePath := fmt.Sprintf("/lib/modules/%s/modules.builtin", kernelVersion)
 	b, err = ioutil.ReadFile(builtinModsFilePath)
 	if err != nil {
-		glog.Errorf("Failed to read file %s with error %v", builtinModsFilePath, err)
+		glog.Warningf("Failed to read file %s with error %v. You can ignore this message when kube-proxy is running inside container without mounting /lib/modules", builtinModsFilePath, err)
```

Comment:
@m1093782566 The file doesn't exist when kube-proxy is running inside container without mounting /lib/modules. Returning a warning instead of an error may be appropriate. Like https://github.com/kubernetes/kubernetes/blob/master/pkg/proxy/ipvs/proxier.go#L467